### ISSUE TITLE
Evaluate mobile and desktop colors

### ISSUE THREAD

**Issue by milafrerichs** [codeforamerica/city-analytics-dashboard#58]
Right now we have two blue colors. 
To get more visibility we discussed to use complementary colors.

#### Current colors
![bildschirmfoto 2015-08-19 um 15 50 11](https://cloud.githubusercontent.com/assets/688980/9494241/33e3c0f4-4c05-11e5-821c-91d91eda362f.png)

#### Two suggestions:
![bildschirmfoto 2015-08-25 um 17 38 34](https://cloud.githubusercontent.com/assets/688980/9494238/30cf5c52-4c05-11e5-801d-4291772f6733.png)
![bildschirmfoto 2015-08-25 um 17 37 22](https://cloud.githubusercontent.com/assets/688980/9494247/3991737a-4c05-11e5-81be-861345a2c3f9.png)

**Comment by migurski** [comment]
@phae: can we get your eyes on this? We are adding a contrasting color to replace one of the two blues; it will also appear in labels like these from #59:

![label colors](https://cloud.githubusercontent.com/assets/688980/9497249/6b6b3fc0-4c14-11e5-93fb-4715c476a254.png)

The two options above happen to be right in my zone of greatest colorbindness, and it’s hard for me to distinguish between them. I’d love you to weigh in on color choices overall.

**Comment by phae** [comment]
Eyeing...

The fact that it's a problem for your colour-blindness is an issue of its own.  :(  Ideally I'd want the one that you can see best, @migurski - are either of the two more clearly defined to you?

If they're equally viewable, then my preference is to the first of the two (the darker shade of the orange).  The two shades of blue side by side on the graph are definitely harder to see in difference and I'm not feeling the light shade of yellowy orange. 

The blue/orange combo coincidentally matches the colour pairing combo we're using on Chime, which is a nice nod towards a unified look and feel for internal-CfA-made apps.

In case you're interested, our shades for Chime are: #2b93d1 (cyan blue), #bfe4f9 (pale blue) and #f26522 (orange), if you want to try those out instead.

**Comment by migurski** [comment]
The first of the two looks a bit less muddy to me. The contrast with the blue is fine in both, but the second uses a shade of color that I know I’m weak in, so it reads as muddy. I’d be curious to hear what @ericfischer thinks, I know he’s colorblind in a different way than I am.

**Comment by phae** [comment]
The second one looks "muddy" to me too in terms of colour cleanliness, interestingly. It's a very desaturated shade of orange.

**Comment by ericfischer** [comment]
Thanks for asking. Either of the two-color stacked bars is good for me, with the first being a little better.

**Comment by milafrerichs** [comment]
I created a different set of colors. Base color is now the darker desktop color and the complementary color for it is the brown:
![bildschirmfoto 2015-09-01 um 11 52 28](https://cloud.githubusercontent.com/assets/688980/9601131/f2382dbc-509f-11e5-8ac8-d513dc94586c.png)

**Comment by ericfischer** [comment]
From a deuteranope perspective this is much muddier than the previous orange, by the way, although still possible to see.

**Comment by migurski** [comment]
Yes, I think based on this morning’s conversation we’re going with one of the snappier options above that has a larger value difference.
